packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -411,15 +411,15 @@ function checkFilesFromTsConfig(packageName: string, tsconfig: TsConfig, directo
     if (file.startsWith("./")) {
       throw new Error(`In ${tsconfigPath}: Unnecessary "./" at the start of ${file}`);
     }
-    if (!isRelativePath(file)) {
+    if (!isRelativePath(file.replace(/^(?:\.\.\/)+/, ""))) {
```

Comment:
Doesn’t this allow tests from _anywhere_? Or is there something else that would prevent `types/react-dom/tsconfig.json` from listing `../react/react-tests.d.ts`?